packages/e2e-tests/specs/block-deletion.test.js

```diff
@@ -20,10 +22,17 @@ const addThreeParagraphsToNewPost = async () => {
 	await page.keyboard.press( 'Enter' );
 };
 
-const clickOnBlockSettingsMenuItem = async ( buttonLabel ) => {
-	await expect( page ).toClick( '.block-editor-block-settings-menu__toggle' );
-	const itemButton = ( await page.$x( `//*[contains(@class, "block-editor-block-settings-menu__popover")]//button[contains(text(), '${ buttonLabel }')]` ) )[ 0 ];
-	await itemButton.click();
+/**
+ * Note Puppeteer has a problem clicking on items in the Popover that require
+ * scrolling the Popover to be visible. To work around this we use the keyboard
+ * to select the "Remove Block" button
+ */
```

Comment:
To be clear, the problem in not puppeteer, but the scrollbars are not showing and the popover is not scrollable for some reason. (It's an actual bug somewhere)

Reply:
I'll tweak the comment a little. I'll leave the context though as it's helpful.

Reply:
I've updated the comment